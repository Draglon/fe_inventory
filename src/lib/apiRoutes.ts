// ---User (SELF)---
export const authLoginRoute = "/auth/login";
export const authRegistrationRoute = "/auth/register";
export const authUserRoute = "/auth/user";
