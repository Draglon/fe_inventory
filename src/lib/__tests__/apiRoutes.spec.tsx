import * as apiRoutes from "../apiRoutes";

it("apiRoutes constants match snapshot", () => {
  expect(apiRoutes).toMatchSnapshot();
});
