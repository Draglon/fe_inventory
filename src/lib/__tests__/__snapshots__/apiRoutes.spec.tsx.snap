// Jest Snapshot v1, https://jestjs.io/docs/snapshot-testing

exports[`apiRoutes constants match snapshot 1`] = `
{
  "authLoginRoute": "/auth/login",
  "authRegistrationRoute": "/auth/register",
  "authUserRoute": "/auth/user",
}
`;
